src/pages/AddPersonalBankAccountPage.js

```diff
@@ -76,7 +78,11 @@ class AddPersonalBankAccountPage extends React.Component {
      * @param {Object} values - form input values passed by the Form component
```

Comment:
NAB since this wasn't modified in this PR and I'm not sure if all these are mandatory, the JSDocs style guide says to _Document object parameters with separate lines e.g. @param {Object} parameters followed by @param {String} parameters.field_